src/NoteWorkspace.ts

```diff
@@ -75,6 +75,7 @@ export class NoteWorkspace {
   static _rxWikiLink = '\\[\\[[^sep\\]]+(sep[^sep\\]]+)?\\]\\]'; // [[wiki-link-regex(|with potential pipe)?]] Note: "sep" will be replaced with pipedWikiLinksSeparator on compile
   static _rxTitle = '(?<=^( {0,3}#[^\\S\\r\\n]+)).+';
   static _rxMarkdownWordPattern = '([_\\p{L}#\\.\\/\\\\]+)'; // had to add [".", "/", "\"] to get relative path completion working and ["#"] to get tag completion working
+  static _rxMarkdownHyperlink = '\\[[^\\[\\]]*\\]\\((?!https?)[^\\(\\)\\[\\] ]+\\)';
```

Comment:
Can you add some hyperlink tests to https://github.com/kortina/vscode-markdown-notes/blob/211ca0ac0f5fb89a9315da04ee7141e5e0ef6cf6/src/test/jest/extension.test.ts#L127 to make sure `_rxWikiLink` does not match a hyperlink? 

And then add a new test set for `_rxMarkdownHyperlink` with examples you do / don't want to match?